lib/gumboot/shared_examples/foreign_keys.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.shared_examples 'Foreign Keys' do
+  conn = ActiveRecord::Base.connection
+  context 'Permissions Foreign Keys', if: conn.supports_foreign_keys? do
+    let(:permission_fk) do
+      conn.foreign_keys(:permissions).find do |fk|
+        fk.options[:column] == 'role_id'
+      end
+    end
+    it 'should have a foreign key' do
+      expect(permission_fk).to_not be_nil
+    end
+  end
```

Comment:
Seems like there should be additional specs to ensure the correct 'to' table? In this case for example `role_id` points to `role`. Is that doable?